routes/index.js

```diff
@@ -356,6 +357,19 @@ router.post('/register-admin', (req, res) => {
     // let user know that access is denied, do not let them know there is a host requirement
     res.status(500).json("access denied");
   }
-})
+});
+
+/* GET uuid-gen - generate a uuid, store it in list, send it back to user */
+router.get('/uuid-gen', (req, res) => {
+  var uuidStr = uuid.GenerateUUID();
+  uuid.addUUIDToList(uuidStr);
+  req.flash('success', uuidStr);
```

Comment:
Maybe flash the resultant URL instead of just the UUID?

Reply:
i will do that until the url has been decided